symbology/create-symbol-styles-from-web-styles/src/main/java/com/esri/samples/create_symbol_styles_from_web_styles/CreateSymbolStylesFromWebStylesSample.java

```diff
@@ -193,34 +187,29 @@ public void start(Stage stage) {
    */
   private void setupLegend() {
 
-    // create a grid pane and set the size, background color and spacing
+    // create a box to show the legend
+    vBox = new VBox();
+    vBox.setMaxSize(220, 300);
+    vBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(255, 255, 255 ,0.5)"), CornerRadii.EMPTY,
+      Insets.EMPTY)));
+    vBox.setPadding(new Insets(10.0));
+    vBox.setAlignment(Pos.TOP_CENTER);
+
+    // create a grid pane
     legendGridPane = new GridPane();
-    legendGridPane.getColumnConstraints().addAll(Arrays.asList(new ColumnConstraints(70), new ColumnConstraints(120)));
+    legendGridPane.getColumnConstraints().add(new ColumnConstraints(50));
     legendGridPane.setPadding(new Insets(10));
     legendGridPane.setMaxWidth(175);
     legendGridPane.setVgap(12);
 
-    headerGridPane = new GridPane();
-    headerGridPane.getColumnConstraints().addAll(Arrays.asList(new ColumnConstraints(70), new ColumnConstraints(120)));
-    headerGridPane.setMaxWidth(175);
-    headerGridPane.setMaxHeight(570);
-    headerGridPane.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(255,255,255, 0.9)"), CornerRadii.EMPTY,
-      Insets.EMPTY)));
-    headerGridPane.setPadding(new Insets(10));
-    headerGridPane.setVgap(12);
-
     // create a label to display the symbol style name as the title of the legend and add to the grid pane
     Label legendHeader = new Label("Style: " + symbolStyle.getStyleName());
-    legendHeader.setStyle("-fx-font-weight: bold");
-    headerGridPane.add(legendHeader, 0, 0,2,1);
-
-    // create labels for the column headings and add to the grid pane
-    Label symbolHeader = new Label("Symbol");
-    symbolHeader.setStyle("-fx-font-weight: bold");
-    Label nameHeader = new Label("Name");
-    nameHeader.setStyle("-fx-font-weight: bold");
-    headerGridPane.add(symbolHeader, 0, 1);
-    headerGridPane.add(nameHeader, 1, 1);
+    legendHeader.setStyle("-fx-font-size: 15; -fx-font-weight: bold;");
```

Comment:
```suggestion
    legendHeader.setStyle("-fx-font-size: 12; -fx-font-weight: bold;");
```
Means the whole web symbol style name is visible without an ellipsis.

Reply:
i've increased the vbox width slightly instead for readability: do you still see an ellipsis with the vbox wider on Windows?

Reply:
Spacing looks good but yes still see an ellipsis. Width of 250 seems to be fine.

230:
![image](https://user-images.githubusercontent.com/56556822/137177074-df44fdc3-f704-4e5d-8180-ee510d4f3070.png)

250:
![image](https://user-images.githubusercontent.com/56556822/137177151-a3be4550-dd69-4375-9c9d-74257bf50417.png)

Reply:
updated to 250, thanks!